src/sap/fhir/model/r4/FHIRUtils.js

```diff
@@ -717,6 +717,42 @@ sap.ui.define([
 		return sFullUrl;
 	};
 
+	/**
+	* Extracts resource IDs from the error description in FHIR OperationOutcome.
+	* @param {Array<object>} aOperationOutcomes - The array of FHIR OperationOutcome objects.
+	* @returns {Array<string>} - An array containing the extracted resource IDs.
+	* @private
+	* @since 2.3.8
+	*/
+	FHIRUtils.getsIdFromOperationOutcome = function(aOperationOutcomes){
```

Comment:
this neednot be in plural.. a already signifies array .. so aOpertaionOutcome would be sufficient